tensorrt_llm/_torch/pyexecutor/resource_manager.py

```diff
@@ -155,18 +155,33 @@ def __init__(
                 (num_kv_heads + tp_size - 1) // tp_size
                 for _ in range(self.num_local_layers)
             ]
+            self.total_num_kv_heads_per_layer = [
```

Comment:
What is the difference between `total_num_kv_heads_per_layer` and `num_kv_heads_per_layer`? Is one of them for the layers that each partition is responsible for.